src/main/java/org/roy/loadx/transaction/TransactionPrinter.java

```diff
@@ -33,7 +33,7 @@ public String getCount() {
 		static private final String TRANSACTION_HEADER = "Transaction";
 		static private final String AVERAGE_HEADER = "Average";
 		static private final String COUNT_HEADER = "Count";
-		static private final int MAX_ROW_WIDTH = 78;
+		static private final int MAX_ROW_WIDTH = 119;
```

Comment:
Learned something. Deserves a comment or better a MAX_ROW_WIDTH and a CONSOLE_MAX_ROW_WIDTH = MAX_ROW_WIDTH - 1; // -1 to account for wrapping on end of line character